nise/generators/oci/oci_constants.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCI Report Constants"""
+from faker import Faker
+
+
+class OCIReportConstantColumns:
```

Comment:
This looks like a good candidate for a frozen dataclass.

Reply:
One caveat is that attributes on frozen data classes are immutable, but not the values. Since `oci_regions_to_domain` is mutable, in order to make _that_ frozen, which is a good idea for global constants, that too will need to be a frozen dataclass, and so on.